source/developers/dotnet/quickstart.md

```diff
@@ -0,0 +1,212 @@
+To run the following example, click on [Link] and start the project
+```cs
+using System;
+using Minio;
+using Minio.Exceptions;
+using Minio.DataModel;
+using System.Threading.Tasks;
+
+namespace FileUploader
+{
+    class FileUpload
+    {
+        static void Main(string[] args)
+        {
+            var endpoint  = "play.min.io";
+            var accessKey = "Q3AM3UQ867SPQQA43P2F";
+            var secretKey = "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG";
+            try
+            {
+                var minio = new MinioClient()
+                                    .WithEndpoint(endpoint)
+                                    .WithCredentials(accessKey,
+                                             secretKey)
+                                    .WithSSL()
+                                    .Build();
+                FileUpload.Run(minio).Wait();
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            Console.ReadLine();
+        }
+
+        // File uploader task.
+        private async static Task Run(MinioClient minio)
+        {
+            var bucketName = "mymusic";
+            var location   = "us-east-1";
+            var objectName = "golden-oldies.zip";
+            var filePath = "C:\\Users\\username\\Downloads\\golden_oldies.mp3";
+            var contentType = "application/zip";
+
+            try
+            {
+                // Make a bucket on the server, if not already present.
+                bool found = await minio.BucketExistsAsync(bucketName);
+                if (!found)
+                {
+                    await minio.MakeBucketAsync(bucketName, location);
+                }
+                // Upload a file to bucket.
+                await minio.PutObjectAsync(bucketName, objectName, filePath, contentType);
+                Console.WriteLine("Successfully uploaded " + objectName );
+            }
+            catch (MinioException e)
+            {
+                Console.WriteLine("File Upload Error: {0}", e.Message);
+            }
+        }
+    }
+}
+```
+
+## Running MinIO Client Examples
+#### On Windows
+* Clone this repository and open the Minio.Sln in Visual Studio 2017.
+
+* Enter your credentials and bucket name, object name etc.in Minio.Examples/Program.cs
+  Uncomment the example test cases such as below in Program.cs to run an example.
+```cs
+  //Cases.MakeBucket.Run(minioClient, bucketName).Wait();
+```
+* Run the Minio.Client.Examples project from Visual Studio
+#### On Linux
+
+##### Setting .NET SDK on Linux (Ubuntu 21.10)
+<blockquote> NOTE: minio-dotnet requires .NET 5.x SDK to build on Linux. </blockquote>
+
+* Install [.Net SDK](https://docs.microsoft.com/en-us/dotnet/core/install/linux?WT.mc_id=dotnet-35129-website)
+
+```
+wget https://packages.microsoft.com/config/ubuntu/21.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```

Comment:
```suggestion
wget https://packages.microsoft.com/config/ubuntu/21.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```

this is not pointing to 21.10 as per the title. 

My suggestion go to 22.04 LTS here, also we can remove the blockquote minio-dotnet actually compiles fine even with .Net 2.1

Perhaps point to this https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#2204